flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/connector/datagen/table/datagen/DataGenTableSource.java

```diff
@@ -16,10 +16,11 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.factories.datagen;
+package org.apache.flink.connector.datagen.table.datagen;
```

Comment:
`datagen.table.datagen`?

Reply:
Let's just say this PR is still in draft mode and I definitely would've seen this still. :-)